src/pouchdb/platformReactNative.storage.ts

```diff
@@ -0,0 +1,14 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+export const storage = {
+  getItem: async (key: string): Promise<string | null> => {
+    return AsyncStorage.getItem(key)
+  },
+  setItem: async (key: string, value: string | undefined): Promise<void> => {
+    if (value === undefined) return
+    return AsyncStorage.setItem(key, value)
+  },
+  removeItem: async (key: string): Promise<void> => {
+    return AsyncStorage.removeItem(key)
+  }
```

Comment:
A prefix is added to the key by PouchDB ?

Reply:
No, the keys will be:
- `cozy-client-pouch-link-lastreplicateddocid`
- `cozy-client-pouch-link-lastreplicationsequence`
- `cozy-client-pouch-link-synced`